frontend/package.json

```diff
@@ -13,12 +13,16 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.8.1",
-    "styled-components": "^5.3.6"
+    "styled-components": "^5.3.8"
```

Comment:
Mostly just curious but when you ran an npm install did it update this automatically or did you intentionally increment the version of styled components?